controllers/home-routes.js

```diff
@@ -75,6 +75,43 @@ router.get("/group/:id", async (req, res) => {
   });
 });
 
+// add new group
+router.post('/', async (req, res) => {
+  try {
+    const newGroup = await Group.create({
+      where: {
+        title: req.params.title,
+        user_id: req.session.user_id,
+      }
+    });
+
+    res.status(200).json(newGroup);
+  } catch(err) {
+    res.status(400).json(err);
+  }
+});
+
+// delete group
+router.delete('/group/:id', async (req, res) => {
```

Comment:
We need to add code to delete relevant GroupUser and GroupList data. Right now there's a leak.